scss/contacts/action-group.scss

```diff
@@ -0,0 +1,29 @@
+@import "../variables";
+@import "../mixins";
+
+.action-group-wrapper button {
```

Comment:
Do we want to be targeting all `button` in `.action-group-wrapper`, or do we want to be targetting `.action-group-icon`?